pkg/controller/operator.go

```diff
@@ -185,8 +188,11 @@ func (o *KubernikusOperator) Run(stopCh <-chan struct{}, wg *sync.WaitGroup) {
 	o.Factories.Kubernikus.WaitForCacheSync(stopCh)
 	o.Factories.Kubernetes.WaitForCacheSync(stopCh)
 
-	o.Logger.Log("msg", "Cache primed. Ready for Action!")
-
+	o.Logger.Log("msg", "Cache primed")
+	if _, found := o.Config.Kubernikus.Controllers["migration"]; found {
```

Comment:
Is this really necessary here? Wouldn't the immediate first reconciliation be equivalent?

Reply:
The idea was to have the klusters migrated before any controllers are running to avoid running controllers that expect the new model on an old model that hasn't been migrated yet.
Also to not accidentally modify models that haven't been migrated yet (so that we don't loose the old data we might need for the migration) we need to make sure they are updated to be `Disabled()` before other controllers run.

Reply:
Ah, ok. Good. Carry on... :)